### DISCUSSION TITLE
Implement SOFORT gateway type for Stripe

### DISCUSSION THREAD

**Discussion by andris-sevcenko** [craftcms/commerce-stripe#165] [Ideas]
Implementing a SOFORT gateway will allow sites to accept payments using the SOFORT method.

Related: #25

**Comment by nordentwickler** [comment]
And it would be awesome if "Giropay" is also possible.

**Comment by andris-sevcenko** [comment]
@nordentwickler yep, #27

**Comment by andris-sevcenko** [comment]
As we gear up for [EU SCA regulation](https://stripe.com/guides/strong-customer-authentication) coming in effect on Sep 14th, 2019, we have to assess where we stand on things. The SCA regulations would make implementing this in the current state obsolete in September. 

On our end, it means that implementation of this depends [on the entry for SOFORT](https://stripe.com/docs/payments/payment-methods#transitioning) changing from "Planned" to "Supported".

It shouldn't take long, because that's a bunch of customers that Stripe doesn't want to lose on September, either.
